src/Textarea/Textarea.tsx

```diff
@@ -44,10 +44,11 @@ type StyledTextareaProps = React.ComponentPropsWithRef<"textarea"> &
     errorList?: string[];
     error?: boolean;
     rows?: number;
+    isResizeable?: boolean;
   };
 const StyledTextarea: React.SFC<StyledTextareaProps> = styled.textarea(
   space,
-  ({ theme }) => ({
+  ({ theme, isResizeable }: any): CSSObject => ({
```

Comment:
I am wondering if we could've used `StyledTextareaProps` instead of `any` here to make use of TS's type checking if someone accidentally puts a prop there that is not part of `StyledTextareaProps`?

Reply:
Oops sorry i missed this comment. Yes good call. We can push another release for that.